tests/unittests/triton/test_model_to_triton.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from pathlib import Path
+
+import torch
+import torchvision
+from towhee.runtime.node_config import NodeConfig
+from towhee.serve.triton.model_to_triton import ModelToTriton
+
+
+public_path = Path(__file__).parent.parent.resolve()
```

Comment:
`from tempfile import TemporaryDirectory `
Use this lib to create a temp dir, it makes sure that temp dir will be deleted